src/applications/gi-sandbox/containers/search/ResultCard.jsx

```diff
@@ -53,18 +54,34 @@ export function ResultCard({
     programLengthInHours,
   } = institution;
   const compareChecked = !!compare.search.institutions[facilityCode];
+  const compareLength = compare.search.loaded.length;
   const handleCompareUpdate = e => {
+    recordEvent({
+      event: `${compareLength + 1} schools in compare`,
+    });
     if (e.target.checked && !compareChecked) {
-      if (compare.search.loaded.length === 3) {
+      if (compareLength === 3) {
         dispatchShowModal('comparisonLimit');
       } else {
         dispatchAddCompareInstitution(institution);
+        recordEvent({
+          event: `Added ${institution.name} to compare`,
+        });
       }
     } else {
       dispatchRemoveCompareInstitution(facilityCode);
+      recordEvent({
+        event: `Removed ${institution} from compare`,
+      });
     }
   };
 
+  // const handleCompareClick = () => {
```

Comment:
Doesn't look like this handler is being used.

Reply:
Missed this one just pushed an update removing the commented handler. Thanks!